packages/react-reconciler/src/ReactFiberWorkLoop.new.js

```diff
@@ -1865,6 +1864,22 @@ function commitRootImpl(root, renderPriorityLevel) {
     // times out.
   }
 
+  // If there are pending passive effects, schedule a callback to process them.
+  // Do this as early as possible, before anything else in the commit phase
+  // is scheduled.
```

Comment:
"Do this as early as possible"

Why?

Reply:
#16714

"...so it is queued before anything else that might get scheduled in the commit phase" Better comment?

Reply:
Ah, I wasn't aware of #16714. Just found myself wondering about the "why" since this comment was more about the "what"